test/noneP.js

```diff
@@ -0,0 +1,72 @@
+import * as RA from '../src';
+import eq from './shared/eq';
+
+const wait = (ms = 0) => new Promise(res => setTimeout(res, ms));
+
+describe('noneP', function() {
+  context('given all promises reject', function() {
+    specify('should resolve with the reasons', async function() {
+      const reasons = await RA.noneP([
+        RA.rejectP(1),
+        RA.rejectP('b'),
+        RA.rejectP(),
+      ]);
+      eq(reasons, [1, 'b', undefined]);
+    });
+  });
+
+  context('given one promise resolves', function() {
```

Comment:
Thinking what is more readable:
```js
  context('given one promise resolves', function() {
    specify('should reject with the fulfillment value', async function() {
      const value = await RA.noneP([
        RA.rejectP(),
        RA.rejectP(),
        RA.resolveP('oops'),
      ]).catch(RA.resolveP);
      eq(value, 'oops');
    });
  });
```
vs
```js
  context('given one promise resolves', function() {
    specify('should reject with the fulfillment value', async function() {
      try {
        const value = await RA.noneP([
          RA.rejectP(),
          RA.rejectP(),
          RA.resolveP('oops'),
        ]);
        throw new Error('should reject');
      }
      catch(e) {
         assert.strictEqual(e.message, 'oops');
      }
    });
  });
```
I'll leave the decision up to you, but if you do go for your original version pls divide the the definition of value const and assertion with one empty line. 

```js
  context('given one promise resolves', function() {
    specify('should reject with the fulfillment value', async function() {
      const value = await RA.noneP([
        RA.rejectP(),
        RA.rejectP(),
        RA.resolveP('oops'),
      ]).catch(RA.resolveP);

      eq(value, 'oops');
    });
  });
```